node.gyp

```diff
@@ -215,6 +222,37 @@
 
 
       'conditions': [
+        [ 'node_shared=="false"', {
+          'msvs_settings': {
+            'VCManifestTool': {
+              'EmbedManifest': 'true',
+              'AdditionalManifestFiles': 'src/res/node.exe.extra.manifest'
+            }
+          },
+        }, {
+          'defines': [
+            'NODE_SHARED_MODE',
+          ],
+          'conditions': [
+            [ 'node_module_ver!=""', {
+              'product_extension': 'so.<(node_module_ver)',
+            }]
+          ],
+        }],
+        [ 'node_no_bundled_v8=="false"', {
+          'include_dirs': [
+            'deps/v8', # include/v8_platform.h
```

Comment:
It exists in the node.gyp in master too.

It makes sense to include it because I don't think it would be clear to a casual reader why deps/v8 rather than deps/v8/include should be on the include path.

It's because some headers `#include "include/v8_platform.h"` instead of `#include "v8_platfom.h"`.  That's fixed in V8 5.1 but we can't remove it until we upgrade.